CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## next
+
+### Other Changes
+
+* switch to jest testing framework ([#7](https://github.com/jfairbank/run-elm/pull/7))
```

Comment:
I don't think it's important to include this in the CHANGELOG since it's internal.

Reply:
OK, I won't argue :–) Removing that 👍